pandas/core/panel.py

```diff
@@ -1235,7 +1235,12 @@ def reindex(self, *args, **kwargs):
         kwargs.update(axes)
         kwargs.pop('axis', None)
         kwargs.pop('labels', None)
-        return super(Panel, self).reindex(**kwargs)
+
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", FutureWarning)
+            # do not warn about constructing Panel when reindexing
+            result = super(Panel, self).reindex(**kwargs)
+        return result
```

Comment:
I had no intention to add this warning filter here, but otherwise I cannot test the `raise_conflict` deprecation for `Panel`, as this warning was throwing off me trying to catch another warning. If we don't care about testing that kwarg deprecation for `Panel`,  I can revert this change here.